pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mmcif"
+description="mmCIF Core Access Library"
+readme = "README.md"
+requires-python = ">= 3.6"
+license = {text = "Apache-2.0"}
+authors = [
+ { name = "John Westbrook", email = "john.westbrook@rcsb.org" }	
+]
+classifiers = [
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3.14",  
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Natural Language :: English"
+]
+
+dynamic = ["dependencies", "version", "optional-dependencies"]
+
+[tool.setuptools]
+zip-safe = false
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["requirements.txt"]}
+version = { attr = "mmcif.__version__" }
+optional-dependencies.test = { file = ["requirements-test.txt"] }
+optional-dependencies.docs = { file = ["requirements-test.txt", "requirements-doc.txt"] }
+optional-dependencies.all = { file = ["requiremnets.txt", "requirements-test.txt"] }
+
+[tool.setuptools.packages.find]
+exclude = ["mmcif.tests", "tests.*"]
+namespaces = false
+
+[tool.setuptools.package-data]
+mmcif = ["*.md", "*.rst", "*.txt", "*.h", "*.C", ".c", "*.cpp"]
```

Comment:
I don't know about you, but I really struggled to get this part to work as desired. Have you tried building the package locally and checking the contents of the source distribution tarball? If not, I'd suggest doing so and comparing it to what's currently provided on PyPI: https://pypi.org/project/mmcif/#files.

I ended up having to add certain things to the MANIFEST (which it looks like you do) and including the explicit `tool.setuptools.packages.find.include` [setting](https://github.com/rcsb/py-rcsb-api/blob/master/pyproject.toml#L53-L59). I'm still not sure if the `tool.setuptools.package-data` setting I have is actually doing anything or not.

Reply:
I had to do the following to get it to include files in the package: https://github.com/rcsb/py-rcsb-api/blob/master/pyproject.toml#L53-L59